src/components/pages/PowerheadPage/view.js

```diff
@@ -40,20 +42,21 @@ function renderPeopleStatsList(report, monthIndex) {
   const extFteVal = Math.ceil(report.ext_fte[monthIndex]);
   return (
     <ul className={styles.peopleStats}>
-      {renderPeopleStatsItem('Total exts.', extFteVal, 'FTE', true)}
-      {renderPeopleStatsItem('Total ints.', totalIntsVal, 'FTE')}
-      {renderPeopleStatsItem('Booked', bookedVal, 'FTE')}
-      {renderPeopleStatsItem('Bench', benchVal, 'FTE')}
+      {renderPeopleStatsItem('Total exts.', extFteVal, 'FTE', 'exts', true)}
+      {renderPeopleStatsItem('Total ints.', totalIntsVal, 'FTE', 'ints')}
+      {renderPeopleStatsItem('Booked', bookedVal, 'FTE', 'booked')}
+      {renderPeopleStatsItem('Bench', benchVal, 'FTE', 'bench')}
     </ul>
   );
 }
 
-function renderFinancialsStatsItem(label, value, legendBarStyle, special = false) {
```

Comment:
But you don't need to update this PR. Let's merge this and do the refactor separately in another PR.